CHANGELOG.md

```diff
@@ -15,6 +15,9 @@
 - Update local help (2024-06-19)
 - refactor: safer types #3993
 - keep aspect ratio in quoted images #3999
+- Update `@deltachat/stdio-rpc-server` and `deltachat/jsonrpc-client` to `1.141.2`
+  - fix memory leak in jsonrpc client code (to be specific in yerpc)
+  - support vcards exported by protonmail
```

Comment:
i think, "Display vCard contact name in the message summary" is worth mentioning as well.
maybe also some fixes from >1.140.2

Reply:
Maybe "Case-insensitive search for non-ASCII messages" also

Reply:
since @farooqkz already merged this, I added those in a new pr in https://github.com/deltachat/deltachat-desktop/pull/4026